components/clp-package-utils/pyproject.toml

```diff
@@ -7,6 +7,7 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.8 || ^3.10"
+pymongo = "^4.6.1"
```

Comment:
Should go after `msgpack` to stay alphabetical. (The reason `python` goes first is just because it's different than the package versions and I felt like putting it first made more sense.)